floyd/cli/data_upload_utils.py

```diff
@@ -71,19 +67,21 @@ def initialize_new_upload(data_config, access_token):
 
     # Create data object using API
     data = DataRequest(name=data_name,
-                       description=version,
-                       data_type='gzip',
-                       version=version)
-    data_id = DataClient().create(data)
-    if not data_id:
+                       # TODO: support commit message
```

Comment:
Good catch. To keep it similar to expt, we can add a command line parameter called `--message`